packages/checkout-com/README.md

```diff
@@ -6,51 +6,101 @@
 ['@vue-storefront/checkout-com/nuxt', { publicKey: 'pk_test_your-public-key' }],
 ```
 
-## Credit card component
-1. Import `useCkoCard`:
+## Render payment handlers
+1. Import `useCko` and `useCkoCard`:
 ```js
-import { useCkoCard } from '@vue-storefront/checkout-com';
+import { useCko, useCkoCard } from '@vue-storefront/checkout-com';
 ```
 
-2. `useCkoCard` returns:
+2. `useCko` returns:
+```ts
+interface {
+    availableMethods: { name: string, [key: string]: any },
```

Comment:
This collides with the last lines of this README, that say `availableMethods` is just an array of strings.

Reply:
Ok, I fixed it at the bottom of the readme